mypage/serializers.py

```
@@ -158,9 +158,17 @@ class Meta:
     class Meta:
         model = Notification
-        fields = ["id", "message", "isRead", "createdAt"]
\ No newline at end of file
+        fields = [
+            "id",
+            "message",
+            "type",
+            "targetId",
+            "isRead",
+            "createdAt",
+        ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)